data/projects.ini

```diff
@@ -134,8 +134,8 @@ git_url: https://pypi.org/project/bcrypt.git
 description_short: Modern password hashing for your software and your servers
 description_long: Modern password hashing for your software and your servers
 
-[beautifulsoup4]
-version: >=4.7.1
+[bs4]
```

Comment:
Why this change? the name of the project is beautifulsoup4

Reply:
I've undone this because this would break other presets that are using the beautifulsoup4 project